packages/core-js/utils/ast/finders.js

```diff
@@ -1,13 +1,13 @@
 const { findAll } = require('solidity-ast/utils');
 
-function findContractNodeWithId(contractId, asts) {
```

Comment:
Fix naming. It checks on a single AST not on the asts object containing { 'contractNameX': {}, 'contractNameY': {}... }

Reply:
I believe this is not true. Pls look at like 37 in this file.

Reply:
used to work... :( that's why we need test coverage :)